cleanlab_studio/utils/data_enrichment/enrichment_utils.py

```diff
@@ -57,7 +57,7 @@ def get_regex_match(
     response: str, regex_list: List[re.Pattern[str]], disable_warnings: bool
 ) -> Union[str, None]:
     """Extract the first match from the response using the provided regex patterns. Return first match if multiple exist.
-    Note: This function assumes the regex patterns each specify exactly 1 group that is the match group using '(<group>)'."""
+    Note: This function assumes the regex patterns each specify exactly 1 group that is the match group using `'(<group>)'`."""
```

Comment:
```suggestion
    Note: This function assumes the regex patterns each specify exactly 1 group that is the match group using ``'(<group>)'``."""
```

Should this be two since the first edit you made uses two backticks?

Reply:
yes i added